react/src/components/atoms/forms/InputCurrency/index.js

```diff
@@ -224,7 +224,7 @@ const InputCurrency = (props) => {
     language,
     disabled: props.disabled
   };
-  const currency = numbro(inputProps.defaultValue);
+  const currency = inputProps.defaultValue && numbro(inputProps.defaultValue);
```

Comment:
This doesn't allow for a defaultValue of `0`. Maybe:
```suggestion
  const currency = !Number.isNaN(Number(inputProps.defaultValue)) && numbro(inputProps.defaultValue);
```